source/CMakeLists.txt

```diff
@@ -69,12 +69,30 @@ include_directories("${CMAKE_SOURCE_DIR}"
 )
 link_directories(${Boost_LIBRARY_DIRS})
 
-MACRO(ADD_BOOST_TEST EXE_NAME)
+SET(TEST_DEBUG_OUTPUT OFF CACHE BOOL "Shoot test output to stdin")
+
+MACRO(BUILD_BOOST_TEST_ARGS EXE_NAME)
```

Comment:
Just an idea : Maybe it's cleaner to move cmake macro functions in a special directory like, **root_project/cmake** ?
We can use this directory for others functions as well.

Reply:
That's a very good point! If we keep on adding macros here, it would make sense to refactor it your way ;) Let's keep that in mind.

Reply:
:+1: